### ISSUE TITLE
Relatively useless secondary number input

### ISSUE THREAD

**Issue by reneelynn** [SpiderOak/Encryptr#237]
In the Credit Card option, there's an arrow selector to add numbers in addition to the ability to input them directly. The arrow selection is relatively useless the way it works as you can only move up or down sequentially by 1 and I'd opt for removing it.

![screen shot 2015-11-23 at 1 20 12 pm](https://cloud.githubusercontent.com/assets/3867179/11369791/bd25f754-928e-11e5-8500-35dc588bef91.png)

2.0.0RC.13

**Comment by devgeeks** [comment]
Is this on desktop? Great. It didn't used to do that. 

It's because on mobile it is a "number" text box so that a number-preferenced kb pops up.

I can look into fixing it, but it might be better done in a point release after 2.0.0 ships. I would need to make sure that changing it wouldn't mess anything up for entries already added.

**Comment by reneelynn** [comment]
Yep, desktop only, doesn't show in Mobile, and I don't mind having it in a point release. The only one that seriously bothers me for 2.0.0 release is #234

**Comment by devgeeks** [comment]
:+1:
